amp-viewer.js

```diff
@@ -0,0 +1,23 @@
+class AmpViewer {
+ 
+ /**
+  * @param {!Element} hostEl the element to attatch the iframe to.
+  * @param {string} ampUrl the AMP page url.
+  */
+ constructor(hostEl, ampUrl) {
+  /** @type {!Element} */
+  this.hostEl_ = hostEl;
+
+  /** @type {string} */
+  this.ampUrl_ = ampUrl;
+
+  this.attach();
```

Comment:
Generally, given multitude of parameters and customizations, I cannot imagine we can attach iframe right away.